Makefile

```diff
@@ -0,0 +1,75 @@
+MAKEFLAGS += --warn-undefined-variables
+SHELL := /bin/bash
+.SHELLFLAGS := -eu -o pipefail -c
+.DEFAULT_GOAL := help
+
+MYSQL_CONTAINER_NAME := "rt-mysql"
+
+# all targets are phony
+.PHONY: $(shell egrep -o ^[a-zA-Z_-]+: $(MAKEFILE_LIST) | sed 's/://')
+
+--check-docker:
+	@echo ">>> Checking docker engine exists ..."
+	@echo "Server: `docker version --format '{{.Server.Version}}'`"
+	@echo "Client: `docker version --format '{{.Client.Version}}'`"
+	@echo ''
+
+
+--check-poetry:
+	@echo ">>> Checking poetry installed ..."
+	@poetry --version
+	@echo ''
+
+
+db: --check-docker ## Starting MySQL container
+	@echo ">>> Starting MySQL container ..."
+	@docker start ${MYSQL_CONTAINER_NAME} 2> /dev/null || docker run -d	\
+		--name ${MYSQL_CONTAINER_NAME} \
+		-p 3306:3306 \
+		-e MYSQL_DATABASE='rt' \
+		-e MYSQL_ROOT_PASSWORD='root' \
+		bitnami/mysql:latest
+	@echo ''
+	@sleep 10
```

Comment:
note: This should not be fixed value. Need to wait for application starting up with readiness probes with docker containers.

Reply:
Referencing [this answers](https://stackoverflow.com/a/33520390), it is easy & clean way to add `until` wait in `make db` target.

As docker behaviors, the field `.State.Health.Status` seems to be added when we add `--health-*` options with `docker run` command, so we also have to update them.

```shell
# Use mysqladmin command for health checking
--health-cmd "mysqladmin ping" \
--health-interval 10s \
--health-retries 5 \
--health-timeout 5s \
```